### PR TITLE
Fix the 3p JS preload in local dev environment.

### PR DESCRIPTION
**Pull request by lannka**
#4376

### CODE REVIEW

**Comment by erwinmombay** [comment]
@lannka LGTM if you can get it green

File: src/3p-frame.js [hunk 1]
```diff
@@ -165,17 +165,20 @@ export function addDataAndJsonAttributes_(element, attributes) {
...
   preconnect.preload(url, 'document');
+
   // While the URL may point to a custom domain, this URL will always be
   // fetched by it.
-  preconnect.preload(
-      `${urls.thirdParty}/$internalRuntimeVersion$/f.js`, 'script');
+  const scriptUrl = getMode().localDev
+      ? getAdsLocalhost(window) + '/dist.3p/current/integration.js'
+      : `${urls.thirdParty}/$internalRuntimeVersion$/f.js`;
+  preconnect.preload(scriptUrl, 'script');
```
Comment (muxin) [inline src/3p-frame.js]: Does this work in test environment?

Comment (muxin) [inline src/3p-frame.js]: I think in test environment, it should be `http://ads.localhost:9876/base/dist.3p/current/integration.js`

**Comment by lannka** [comment]
@muxin I took a look at the `/base` prefix in test environment. I think it's something we can fix (remove this discrepancy). Filed an issue for that: #4452

**Comment by muxin** [comment]
LGTM

**Pull request merged**